pkg/config/analysis/diag/helper.go

```diff
@@ -24,9 +24,14 @@ var (
 )
 
 type testOrigin struct {
-	name     string
-	ref      resource.Reference
-	fieldMap map[string]int
+	name       string
+	ref        resource.Reference
+	fieldMap   map[string]int
+	isFromFile bool
```

Comment:
Do we have both a property and a function sharing the same name?

Reply:
Fixed, thanks